components/WidgetImage.vue

```diff
@@ -0,0 +1,15 @@
+<script setup lang="ts">
+defineProps<{
+  src: string
+}>()
```

Comment:
```suggestion
type Props = {
  src: string
}

defineProps<Props>()
```

Please type component props as above. Also in other components

Reply:
will do!